frontend/src/components/explorer/mapViewer/SidebarDataPanels.vue

```diff
@@ -160,13 +218,62 @@ export default {
       showMapCardContent: true,
       showSelectionCardContent: true,
       messages,
+      showModal: false,
+      showFullReactionListMissing: false,
+      showFullReactionListMap: false,
+      displayedReaction: 40,
+      limitReaction: 999999999,
```

Comment:
This limit is so high, might as well not have it at all.

Reply:
Maybe we can set it to a smaller number, e.g. 1000? Not sure how the popup window looks like or if it will be slow to load if there are several thousand reactions.

Reply:
One way to test this would be to associate the _Mitochondrion_ map to another compartment through the association file in `data-files`, leading to none of the thousand reactions being correctly associated.
I'm hoping that the loading is fast enough to not require any limit.

Reply:
@nanjiangshu me and Mihail experimented with this on localhost and did not run in to any performance/visual issues. Thinking that is better to remove it for now and add it again if we encounter a problem in the future.